src/fidesapi/routes/generate.py

```diff
@@ -130,16 +146,20 @@ def generate_aws(
     """
     Returns a list of Systems found in AWS.
     """
-    log.info("Setting config for AWS")
-    try:
-        log.info("Generating systems from AWS")
-        aws_systems = generate_aws_systems(
-            organization=organization, aws_config=aws_config
-        )
-    except ConnectorAuthFailureException as error:
-        raise HTTPException(
-            status_code=status.HTTP_403_FORBIDDEN,
-            detail=str(error),
-        )
-
+    log.info("Generating systems from AWS")
+    aws_systems = generate_aws_systems(organization=organization, aws_config=aws_config)
     return [i.dict(exclude_none=True) for i in aws_systems]
+
+
+@route_requires_okta_connector
```

Comment:
top-tier use of language features to add a feature elegantly 💯

Reply:
thanks!